README.md

```diff
@@ -1,8 +1,60 @@
 ## What's this?
 
-This repo holds metadata about the Flutter framework. The metadata is largely
-useful for tooling (the Flutter IntelliJ plugin, the VS Code Dart plugin, ...).
+This repo holds generated metadata about the Flutter framework.
 
-<!-- TODO(devoncarew): Have a describing the various metadata provided -->
+The metadata is useful for Flutter related tooling and is currently used by
+Flutter IDE tools such as the Flutter IntelliJ and VS Code plugins.
 
-<!-- TODO(devoncarew): Have a section listing tools that use this data -->
+If you're using this metadata, please let us know! You can open an issue on the
+repo or add your tool to this readme via a pull request.
+
+## Tools that use this metadata
+
+- the [Flutter IntelliJ] plugin (for IntelliJ and Android Studio)
+- the [Dart-Code] VS Code plugin
+
+## Available metadata
+
+The generated framework data lives at
+https://github.com/flutter/tools_metadata/tree/master/resources.
+
+### Widget Catalog
+
+Available
+[here](https://github.com/flutter/tools_metadata/blob/master/resources/catalog/widgets.json),
+this is the full set of all public widgets from the Flutter framework. It
+includes information like the widget name, it's parent, the library it's defined
+in - like `material`, and the description of the widget.
+
+### Colors
+
+This is a mapping of Flutter framework color names to color values, for both the
+Material and Cupertino sets of colors. It's available in property file and json
+formats.
+
+https://github.com/flutter/tools_metadata/tree/master/resources/colors
+
+### Icons
+
+This is a mapping of:
+
+- font codepoints to icon names, and
+- icon names to file paths for preview icons
+
+This information is available for both the Material and Cupertino icons. In
+addition, each framework icon has a 16x16 png preview image.
+
+https://github.com/flutter/tools_metadata/tree/master/resources/icons
+
+## Working on the repo
+
+### Regenerating metadata
+
+To re-generate the metadata, run `flutter pub run grind generate` from the
```

Comment:
nit: Can probably drop `generate` from the end now it's marked as a default task.

Reply:
I may just keep this as is for the explicitness, but am fine with this changing later so its less wordy.